frontend/src/components/AttendeeExpansionPanel/LmAttendeeAddForm.vue

```diff
@@ -0,0 +1,253 @@
+      <!-- First column -->
+      <div class="d-flex flex-column form-column ga-2">
+        <div class="d-flex align-center ga-4">
+          <v-text-field
+            label="Vorname"
+            variant="outlined"
+            density="comfortable"
+            required
+            :rules="requiredRule"
+            :modelValue="current.firstName"
+            @update:modelValue="current.firstName = $event"
+          ></v-text-field>
+          <v-text-field
+            label="Nachname"
+            variant="outlined"
+            density="comfortable"
+            required
+            :rules="requiredRule"
+            :modelValue="current.lastName"
+            @update:modelValue="current.lastName = $event"
+          ></v-text-field>
+        </div>
+
+        <v-text-field
+          v-if="props.role !== AttendeeRole.HELPER"
+          type="date"
+          v-model="current.birthday"
+          label="Geburtsdatum"
+          required
+          variant="outlined"
+          density="comfortable"
+          :rules="requiredRule"
+        />
+
+        <div class="d-flex align-center ga-4">
+          <v-select
+            :items="tShirtSizes"
+            density="comfortable"
+            variant="outlined"
+            item-title="title"
+            label="T-Shirt-Größe"
+            required
+            :rules="tshirtRules"
+            :modelValue="current.tShirtSize"
+            @update:modelValue="current.tShirtSize = $event"
+          >
+            <template v-slot:selection="{ item }">
+              <v-icon class="mr-4">mdi-tshirt-crew-outline</v-icon>{{ item.title }}
+            </template>
+            <template v-slot:item="{ props }">
+              <v-list-item v-bind="props"></v-list-item>
+            </template>
+          </v-select>
+
+          <v-select
+            :items="foodList"
+            density="comfortable"
+            variant="outlined"
+            item-title="title"
+            label="Essen"
+            required
+            :rules="requiredRule"
+            :modelValue="current.food"
+            @update:modelValue="current.food = $event"
+          >
+            <template v-slot:selection="{ item }">
+              <v-icon class="mr-4">{{ item.props.prependIcon }}</v-icon>
+              {{ item.title }}
+            </template>
+            <template v-slot:item="{ props }">
+              <v-list-item v-bind="props"></v-list-item>
+            </template>
+          </v-select>
+        </div>
+      </div>
+
+      <!-- Second column -->
+      <div class="d-flex flex-column justify-space-between form-column ga-2">
+        <div
+          v-if="[AttendeeRole.YOUTH_LEADER, AttendeeRole.CHILD_LEADER].includes(props.role)"
+          class="d-flex flex-column ga-2"
+        >
+          <v-text-field
+            label="Juleika-Nummer"
+            variant="outlined"
+            density="comfortable"
+            :modelValue="current.juleikaNumber"
```

Comment:
When creating a jugendleiter or kindergruppenleiter the juleikaNumber and juleikaExpireDate are stored, but not shown. But stored at the backend

Reply:
backend bug